mne_bids/write.py

```diff
@@ -1035,9 +1040,15 @@ def write_anat(bids_root, subject, t1w, session=None, acquisition=None,
     if trans is None:
         return anat_dir
 
-    if not isinstance(trans, mne.transforms.Transform):
-        raise ValueError('`trans` must be a "Transform" but is of type "{}"'
-                         .format(type(trans)))
+    if not(isinstance(trans, mne.transforms.Transform) or
+           isinstance(trans, str)):
```

Comment:
`not isinstance(trans, (mne.transforms.Transform, str))`

you can test different valid types with isinstance